dbt/README.md

```diff
@@ -710,18 +710,33 @@ exposes a few options that help to export the right data:
 
 #### Example: Running town close QC reports
 
-We tag the models that comprise our town close QC reports using the `qc_report_town_close`
-tag, and we filter them for a specific township code (like "70") and tax year during export.
+In general, we tag the models that comprise our town close QC reports using the
+`qc_report_town_close` tag. Models with this tag will be included in all town
+close QC reports. If a model should be included in town close reports for tri
+towns but not for non-tri towns, we tag it instead with
+`qc_report_town_close_tri`. Likewise, the `qc_report_town_close_non_tri` tag
+marks models that should be included in town close reports for _non-tri_ towns.
+In both cases, we filter the models for a specific township code (like "70")
+and tax year during export.
 
-Here's an example of how to export those models for a township code defined by `$TOWNSHIP_CODE`
-and a tax year defined by `$TAXYR`:
+We use [selectors](https://docs.getdbt.com/reference/node-selection/yaml-selectors)
+as an interface for exporting tri vs. non-tri reports. Assuming
+a township code defined by `$TOWNSHIP_CODE` and a tax year defined by `$TAXYR`,
+this command will generate town close reports for a **tri town**:
 
 ```
-python3 scripts/export_models.py --select tag:qc_report_town_close --where "taxyr = '$TAXYR' and township_code = '$TOWNSHIP_CODE'"
+python3 scripts/export_models.py --selector select_qc_report_town_close_tri --where "taxyr = '$TAXYR' and township_code = '$TOWNSHIP_CODE'"
 ```
 
-The script will output the reports to the `dbt/export/output/` directory, and will print the
-names of the reports that it exports during execution.
+Use the `select_qc_report_town_close_non_tri` selector to output reports for
+a **non-tri town**:
+
+```
+python3 scripts/export_models.py --selector select_qc_report_town_close_non_tri --where "taxyr = '$TAXYR' and township_code = '$TOWNSHIP_CODE'"
+```
```

Comment:
This new selector interface feels slightly clunky to me: The selector names are quite long, and I feel like it would be more intuitive if the `export_models` script could just infer whether the township is a tri town or a non-tri town based on the taxyr and township code. However, making that change would require either tighter coupling between `export_models` and the town close reports, or (more likely) an additional script like `export_town_close_report` on top of `export_models` that could implement that coupling. It might make sense for us to move in that direction in the future, but for now I went with the most incremental possible change that would satisfy the requirement of splitting tri town close reports from non-tri town reports.

Reply:
Good call! See https://github.com/ccao-data/data-architecture/issues/576.